README.md

```diff
@@ -25,43 +25,49 @@
 
 **Meilisearch JavaScript** is the Meilisearch API client for JavaScript developers.
 
-**Meilisearch** is an open-source search engine. [Discover what Meilisearch is!](https://github.com/meilisearch/meilisearch)
+**Meilisearch** is an open-source search engine. [Learn more about Meilisearch.](https://github.com/meilisearch/meilisearch)
 
 ## Table of Contents <!-- omit in toc -->
 
 - [📖 Documentation](#-documentation)
 - [🔧 Installation](#-installation)
-- [🎬 Getting Started](#-getting-started)
-- [🤖 Compatibility with Meilisearch](#-compatibility-with-meilisearch)
-- [💡 Learn More](#-learn-more)
-- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+- [🎬 Getting started](#-getting-started)
+- [🤖 Compatibility](#-compatibility-with-meilisearch)
+- [💡 Learn more](#-learn-more)
+- [⚙️ Contributing](#️-development-workflow-and-contributing)
 - [📜 API Resources](#-api-resources)
 
 ## 📖 Documentation
 
-See our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+This readme contains all the documentation you need to start using Meilisearch's Javascript SDK.
+
+For general information on how to use Meilisearch, refer to our [main documentation website](https://docs.meilisearch.com/learn/) and the [Meilisearch API Reference](https://docs.meilisearch.com/reference/api/).
```

Comment:
What do we need to precise `main website` ?

Reply:
I'm not 100% sure it is necessary, but in short I am trying to establish the difference between Meilisearch's "main" documentation website (i.e. docs.meilisearch.com) and the SDK documentation (i.e. this readme).

So "main documentation" works in opposition to "This readme contains all the JS SDK documentation".

What do you think?

Reply:
Mmmh, I understand the idea. But once you say `main documentation website` and another time `Meilisearch API reference`  which might lead to maybe more confusion. 
But I do like the idea that we are specific on the fact that we are not talking about the meilisearch-js documentation. Not sure it is necessary though.

I'm not helping here 😂 Sorry

Reply:
You actually are helping quite a bit, don't worry. Those are valid observations.

I'll keep "main documentation" (or something similar), but review the readme so references to both docs.meilisearch and SDK docs are uniform to reduce ambiguity.